### ISSUE TITLE
Reloading doesn't work

### ISSUE THREAD

**Issue by pesterhazy** [mjmeintjes/boot-react-native#56]
With the example app, reloading doesn't work.  I see:

```
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: Reload
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: main.out/node_modules/mattsum.simple_example.core.js
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: main.out/node_modules/mattsum.simple_example.core.js.map
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: main.out/mattsum/simple_example/core.cljs
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: main.out/node_modules/mattsum.simple_example.core.cljs
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: main.out/mattsum/simple_example/core.js.map
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: main.out/mattsum/simple_example/core.js
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]:
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]:
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: ERROR
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: expected: (= 1 2)
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]:   actual: (not (= 1 2))
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]:
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: 1 failures, 0 errors.
Jun 16 16:12:14 Pauluss-MacBook-Air SimpleExampleApp[18993]: JS RELOADING
```

so it looks like it's reloading things, but changing actual code (e.g. adding log messages) doesn't have any effect. I've tried multiple boot-reload versions, but they all don't work.

**Comment by pesterhazy** [comment]
As this used to work fine before, I suspect this issue was caused by https://github.com/mjmeintjes/boot-react-native/pull/46. @samroberton, do you have any idea why this may have broken reloading? Any chance you could try the example app on your machine (it's as easy as `boot dev -pios` or `boot dev -pandroid`)?

**Comment by pesterhazy** [comment]
Also tried `[org.clojars.samroberton/boot-reload       "0.4.9-SNAPSHOT"           :scope  "test"]`, also didn't help

**Comment by pesterhazy** [comment]
Confirmed that reloading works well in the previous version, https://github.com/mjmeintjes/boot-react-native/commits/7f4558b02287e92e9c027f8d6590be53a5958035

**Comment by mjmeintjes** [comment]
Yes, I'm thinking off reverting #46, as although I like the approach, I think the main priority should be to first get boot-react-native working properly with the new versions of React Native. I'm also looking at integrating with RN's hot reloading instead of using boot-reload, but not sure how well that will work.

**Comment by pesterhazy** [comment]
Fixed in master by reverting #46. It'd be great to see that work brought back in the future (if it doesn't impact reloading speed too much).
